spec/index.bs

```diff
@@ -1146,10 +1146,10 @@ Note: Using these IDL attributes is the recommended way of dynamically setting t
 
 <pre class="idl exclude">
 partial interface HTMLScriptElement {
- [CEReactions] attribute ([LegacyNullToEmptyString] DOMString or TrustedScript) innerText;
- [CEReactions] attribute (DOMString or TrustedScript)? textContent;
- [CEReactions] attribute (USVString or TrustedScriptURL) src;
- [CEReactions] attribute (DOMString or TrustedScript) text;
+ [CEReactions] attribute ([LegacyNullToEmptyString] TrustedScript or DOMString) innerText;
```

Comment:
The idl extended attribute needs moving next to the Dom string type iirc (aka after the or

Reply:
Having said that it doesn't appear to be on the original code so I think it might need removing entirely?

Reply:
For reference, the HTMLElement definition: https://html.spec.whatwg.org/#elements-in-the-dom:dom-innertext